pkg/imager/iso/uefi.go

```diff
@@ -167,6 +170,10 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 		}
 	}
 
+	if _, err := cmd.Run("mcopy", "-i", efiBootImg, options.UKISigningCertDerPath, "::EFI/Linux/uki-signing-cert.der"); err != nil {
```

Comment:
is it some standard path? or just dumping something?

Reply:
no standard path, i initially put it under `keys/secureboot` but it seems OVMF doesn;t allow selecting from that path, only `EFI` and `loader` were allowed

Reply:
maybe i can do `loader/keys`

Reply:
loader is I guess for sd-boot loader

Reply:
yeh, but it's `loader/auto`

Reply:
does `EFI/Linux/keys/uki-signing-key.der` makes sense?